index.html

```diff
@@ -1,121 +1,198 @@
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="index.css">
-  <title>Mountain Tranvel</title>
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
-</head>
-<body>
-
-  <!-- THE HEADER -->
-  <header>
-    <h2><a href="#">Mountain Travel By Dcoders</a></h2>
-    <nav>
-      <li><a href="#">Tours</a></li>
-      <li><a href="#">About</a></li>
-      <li><a href="#">Contacts</a></li>
-    </nav>
-  </header>
-
-
-  <!-- THE HERO SECTION -->
-  <section class="hero">
-    <div class="background-image" style="background-image: url('https://res.cloudinary.com/albatrosscourse/image/upload/v1548079444/samples/main.jpg')"></div>
-    <div class="hero-content-area">
-      <h1>Mountain Travel</h1>
-      <h3>Unmissable Adventure Tours Around The World</h3>
-      <a href="#" class="btn">CONTACT US NOW</a>
-    </div>
-  </section>
-
-
-  <!-- THE DESTINATION SECTION -->
-  <section class="destinations"> 
-    <h3 class="title">Some of our destinations:</h3>
-    <p>Tried of ocean? Are plains too plain? Come along with us on one of our mountain adventures. Here are some pictures from people who have had elevated expirences with us.</p>
-    <hr>
-
-    <ul class="grid">
-      <li class="small img" style="background-image: url(https://res.cloudinary.com/albatrosscourse/image/upload/v1548079499/samples/mountain1.jpg)";></li>
-      <li class="large img" style="background-image: url(https://res.cloudinary.com/albatrosscourse/image/upload/v1548079535/samples/mountain2.jpg)";></li>
-      <li class="large img" style="background-image: url(https://res.cloudinary.com/albatrosscourse/image/upload/v1548079563/samples/mountain3.jpg)";></li>
-      <li class="small img" style="background-image: url(https://res.cloudinary.com/albatrosscourse/image/upload/v1548079599/samples/mountain4.jpg)";></li>
-    </ul>
-  </section>
-
-
-  <!-- THE TOUR PAKAGES -->
-  <section class="packages">
-    <h3 class="title">Tour Packages</h3>
-    <p>We offer variety of montaineering packages. Wheather you've climbed Everest or don't even know what a mountain is, we've got the perfect vacation for you.</p>
-    <hr>
-
-    <ul class="grid">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Mountain Tranvel</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans"
+      rel="stylesheet"
```

Comment:
This looks awful! Please remove the meta tags

something like
`<meta> This is good </meta>`

Reply:
This isn't possible